index.js

```diff
@@ -313,7 +320,7 @@ resources.registerAll(Shopify);
  * @private
  */
 function calculateDelay(retryObject) {
-  return maybeRetryMS(retryObject.error) || retryObject.computedValue;
+  return maybeRetryMS(retryObject);
```

Comment:
Do you want me to rename `maybeRetryMS` to `calculateDelay` and remove this?

Reply:
Yes, it makes sense.